package.json

```diff
@@ -5,26 +5,23 @@
   "bin": "./bin/confidant",
   "license": "MPL-2.0",
   "main": "./build/main",
-
   "dependencies": {
     "argparse": "^1.0.2",
     "babel": "^5.6.14",
     "debug": "^2.2.0",
     "glob": "^5.0.12",
     "lodash": "^3.9.3",
-    "mz": "^2.0.0"
+    "mz": "^2.0.0",
+    "nconf": "^0.7.1"
```

Comment:
I think we probably don't need `nconf`. We really just need to copy the stuff from `process.env` into our node script invocation